playbook/spec/pb_kits/playbook/pb_currency/currency_spec.rb

```diff
@@ -13,6 +13,7 @@
   it { is_expected.to define_enum_prop(:size).with_default("md").with_values("sm", "md", "lg") }
   it { is_expected.to define_prop(:symbol).with_default("$").of_type(Playbook::Props::String) }
   it { is_expected.to define_enum_prop(:variant).with_default("default").with_values("default", "light", "bold") }
+  it { is_expected.to define_prop(:abbreviate).with_default(false).of_type(Playbook::Props::Boolean) }
```

Comment:
Do we expect any change in a classname when `abbreviate` prop is being passed?

Reply:
good question. No, because the only thing that is changing is the values of the variables being displayed in the return. The class name will not change

Reply:
Let me rephrase that 'no', I don't think we can expect any change

Reply:
I am also looking at the Jest test in place. I think there are some necessary additions.